.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.sh      eol=lf
```

Comment:
Might need to set `* text=auto` for windows users as well - see [here](https://help.github.com/articles/dealing-with-line-endings/#example)